src/components/Cards/Cards.tsx

```diff
@@ -1,11 +1,59 @@
 import React, { FC } from "react";
-import { Link } from "react-router-dom";
-import { product } from "@constants";
+import { Rate } from "antd";
+import { FoodApi } from "services/FoodService";
+import { HeartOutlined } from "@ant-design/icons";
+import s from "./Cards.module.scss";
 
 const Cards: FC = () => {
+  const { data, isLoading, error } = FoodApi.useFetchAllFreshFoodQuery();
+  console.log(data);
   return (
-    <div>
-      <Link to={product + 1234}>Cards</Link>
+    <div className={s.cards}>
+      <div className={s.cards__item}>
+        {data?.map((el) => (
+          <div className={s.card}>
+            <div className={s.card__img}>
+              <img src={`https://spoonacular.com/cdn/ingredients_500x500/${el.image}`} alt="" />
+            </div>
+            <div className={s.card__body}>
+              <h3 className={s.card__title}>{el.name}</h3>
+              <p className={s.card__simpleText}>{el.description.slice(0, 20).trim().concat("...")}</p>
+              <Rate disabled defaultValue={el.popularity} className={s.card__stars} />
+              <div className={s.card__info}>
+                <p className={s.card_lightText}>
+                  Fresheness <span className={s.card__span}>...</span> {el.fresheness}
+                </p>
+                <p className={s.card_lightText}>
+                  Farm <span className={s.card__span}>...........</span> {el.farm}
+                </p>
+                <p className={s.card_lightText}>
+                  Delivery <span className={s.card__span}>......</span> Europe
+                </p>
+                <p className={s.card_lightText}>
+                  Stock <span className={s.card__span}>..........</span>{" "}
```

Comment:
Isn't it a list ?

Reply:
hm,  I think here I can use unordered list, I rarely use it, so I forget about it, I'll change it now

Reply:
Changed